examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -4719,8 +4719,8 @@ server cluster RadonConcentrationMeasurement = 1071 {
   }
 
   enum MeasurementMediumEnum : ENUM8 {
-    kGas = 0;
-    kLiquid = 1;
+    kAir = 0;
```

Comment:
Did this get changed in the spec?

Reply:
I see it in the spec. Surprised we were allowed to change those names that reduce applicability of the cluster without a revision change or just adding water/air.

Reply:
This enum was new in v1.2, so I didn't think it needed a revision change as its addition was part of this revision. We discussed on slack, just noting here so I can resolve with an explanation for future travellers.